kahuna/public/js/components/gr-syndication-icon/gr-syndication-icon.html

```diff
@@ -1,5 +1,5 @@
 <div title="{{ ctrl.states.syndicationReason }}">
-    <gr-icon class="syndication-status--{{ ctrl.states.syndicationStatus }}">
+    <gr-icon style="font-size: 20px;" class="syndication-status--{{ ctrl.states.syndicationStatus }}">
```

Comment:
We try to avoid using style where possible. There is already an option in gr-icon to set a small font-size (`<gr-icon gr-small>...`), perhaps we could add a `gr-large` attribute which will set a font-size of 20px

Reply:
Hi @andrew-nowak I am having difficulties adding a gr-large attribute to the component, somehow it affects all the gr-icons. However I removed the use of style attribute with class attribute. Is this acceptable?

Reply:
thanks!